subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ivy/IvyMappingToVariantIntegrationTest.groovy

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.resolve.ivy
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+
+import spock.lang.Unroll
+
+class IvyMappingToVariantIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
I believe this whole test can be ditched in favor of the other one. What we do on CI is run the whole test suite with the system property set.

Reply:
correct, just removed that 👍